sensor/common/compliance/node_inventory_handler_test.go

```diff
@@ -67,42 +67,44 @@ func fakeNodeScanV2(nodeName string) *storage.NodeScanV2 {
 	return msg
 }
 
-var _ suite.TearDownTestSuite = (*NodeScanHandlerTestSuite)(nil)
+var _ suite.TearDownTestSuite = (*NodeInventoryTestSuite)(nil)
 
-type NodeScanHandlerTestSuite struct {
+type NodeInventoryTestSuite struct {
```

Comment:
```suggestion
type NodeInventoryHandlerTestSuite struct {
```

Because we are testing _Handler_